Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -98,7 +96,7 @@ private void InitIDs()
 #if UNITY_EDITOR
         void Update()
         {
-            //UpdateSettings();
+            UpdateSettings();
```

Comment:
Don't. Shouldn't be needed as hasn't been uncommented for years. Feel free to delete it though